packages/jest-resolve/src/__tests__/resolve.test.ts

```diff
@@ -345,6 +345,51 @@ describe('findNodeModule', () => {
       );
     });
 
+    test('supports nested pattern', () => {
+      const result = Resolver.findNodeModule('#nested', {
+        basedir: path.resolve(importsRoot, './nested-import/index.cjs'),
+        conditions: ['node', 'require'],
+      });
+
+      expect(result).toEqual(
+        path.resolve(importsRoot, './nested-import/node.cjs'),
+      );
+    });
+
+    test('supports array pattern - resolve to first found', () => {
```

Comment:
this and the 2 tests below can be wrapped in a `describe('supports array pattern')`

Reply:
Sure.